release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -53,6 +53,12 @@ The installer now gathers serial console logs from the bootstrap and control pla
 
 For more information, see xref:../installing/installing-troubleshooting.adoc#installation-bootstrap-gather_installing-troubleshooting[Troubleshooting installation issues].
 
+[id="ocp-4-12-ironic-base-image-rhel-9"]
+==== Ironic container images use RHEL 9 base image
+In earlier versions of {product-title}, Ironic container images used Red Hat Enterprise Linux (RHEL) 8 as the base image. From {product-title} 4.12, Ironic container images use RHEL 9 as the base image. This is to enable support for CentOS Stream 9, Python 3.8, and Python 3.9 in Ironic components. 
```

Comment:
Re: _This is..._ 

Please see the IBM Style Guide's note on [ambiguious pronouns](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references). Also, the I[SG recommends](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#enable) using a more direct term than _enable_. Maybe changing it to something like: "The RHEL 9 base image adds support for..."

WDYT?